string type

int64[] mask_shape
float64[] mask

int64[] boxes_shape
float64[] boxes

int64[] scores_shape
float64[] scores

int64[] labels_shape
int64[] labels